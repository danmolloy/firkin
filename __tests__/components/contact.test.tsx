
describe("<Contact />", () => {
  it("contact-section is in the document", () => {})
})