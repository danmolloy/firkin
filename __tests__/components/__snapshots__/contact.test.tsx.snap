// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactSection /> matches snapshot 1`] = `
<div
  class="px-8 pt-8 pb-16 flex flex-col bg-black text-white"
  data-testid="contact-form"
  id="dark-section"
>
  <h2
    class="text-2xl"
  >
    Contact us
  </h2>
  <form
    action="#"
    class=" flex flex-col md:w-4/5 lg:w-1/2 "
  >
    <div
      class="flex flex-col m-2 "
    >
      <label
        class="form-label "
        for="name-input"
      >
        Name
      </label>
      <input
        class="text-black border border-zinc-400 rounded-md w-full p-1 "
        id="name-input"
        name="name"
        type="text"
        value=""
      />
      <div
        class="h-6"
      />
    </div>
    <div
      class="flex flex-col m-2"
    >
      <label
        class="form-label "
        for="email"
      >
        Email
      </label>
      <input
        class="text-black border border-zinc-400 rounded-md w-full p-1 "
        id="email"
        name="email"
        type="email"
        value=""
      />
      <div
        class="h-6"
      />
    </div>
    <div
      class="flex flex-col m-2 mb-0"
    >
      <label
        class="form-label "
        for="msg-text"
      >
        Message 
      </label>
      <textarea
        class=" text-black border border-zinc-400 rounded-md w-full p-1 "
        id="msg-text"
        maxlength="500"
        multiline="6"
        name="message"
        rows="4"
        type="textarea"
      />
      <div
        class="h-6"
      />
    </div>
    <button
      class="disabled:opacity-30 hover:bg-gray-900 border shadow-sm border-yellow-500 text-yellow-500 m-2 p-1 w-24 rounded self-end"
      id="submit-button"
      type="submit"
    >
      Submit
    </button>
    <div />
  </form>
</div>
`;
